hearth/lib/hearth/http/fields.rb

```diff
@@ -47,7 +46,7 @@ def delete(key)
         @entries.delete(key.downcase)
       end
 
-      # @return [Enumerable<String,Field>]
+      # @return [Enumerable<Field>]
       def each(&block)
         @entries.values.each(&block)
```

Comment:
Is this the right implementation for `each`? or would we want it to be over <String, Field> (as a pair?)

Reply:
Fields will be an array of Field. That was a recent change. It's more usable that way.

Reply:
Headers and trailers iterate keys and values. Field has all the properties. It's slightly different.